Test_Task/main.rb

```diff
@@ -0,0 +1,59 @@
+require 'watir'
+require 'nokogiri'
+require 'json'
+require_relative './transactions'
+
+class WebBanking
+  attr_reader :browser, :page
+  def initialize
+    @browser = Watir::Browser.new :chrome
+  end
+
+  def collect_info
+    sign_in
+    go_to_info
+    tr = Transactions.new
```

Comment:
after 300 lines of code, are you sure you'll remember that `tr` means transactions ?

Reply:
code readability counts